properties/src/main/java/gov/cms/ab2d/properties/model/DataType.java

```diff
@@ -0,0 +1,8 @@
+package gov.cms.ab2d.properties.model;
+
+public enum DataType {
```

Comment:
What's this for? I don't see any uses currently.

Reply:
I took out a bunch of validation in the properties module since it implied specific knowledge of business rules for AB2D which I didn't want. I started adding in more generic validation but it really should be db driven and really, too much to take on for this intermediary step so abandoned the effort. It's actually gone in the latest version.